src/Symfony/Component/Message/README.md

```diff
@@ -0,0 +1,13 @@
+Message Component
+=================
+
+The Message component helps application to send and receive messages to/from other applications or via
```

Comment:
The Message composer helps application send and receive...

Reply:
Did you mean "The Messenger component application send and receive..." ?

Reply:
The Message component helps application send and receive.

You need to remove the `to`